### PR TITLE
Navback fix position

### PR DESCRIPTION
**Pull request by NismathNaifar**

### CODE REVIEW

**Comment by NismathNaifar** [comment]
[Problem]: The toggle Navigation bar close button was not the same in Mockup

[Cause]: incorrect margin on separator line and positioning of NavBack icon

[Fix]: Fixed the margin-top of separator and the positioning of NavBack

File: app/src/components/Sidebar.scss [hunk 1]
```diff
@@ -65,10 +65,10 @@ $sidebar-border-radius: 0px 20px 20px 0px;
       width: 224px;
       height: 0px;
       border-top: 2px solid #949499;
-      margin-top: 48px;
+      margin-top: 58px;
       margin-bottom: 48px;
-      right: 16px;
-      left: 16px;
+      // right: 16px;
```
Comment (nightfallb612) [inline app/src/components/Sidebar.scss:70]: Could you delete comment statement?

File: app/src/components/Sidebar.scss [hunk 2]
```diff
@@ -65,10 +65,10 @@ $sidebar-border-radius: 0px 20px 20px 0px;
...
+      // left: 16px;
```
Comment (nightfallb612) [inline app/src/components/Sidebar.scss:71]: Could you delete comment statement?

**Review by nightfallb612 — CHANGES_REQUESTED** [review]

**Review by nightfallb612 — APPROVED** [review]
Review completed.

**Pull request merged**